src/dune_rules/dune_file.mli

```diff
@@ -330,24 +330,35 @@ module Include_subdirs : sig
     | Include of qualification
 end
 
+module Library_redirect : sig
+  type 'old_name t =
+    { project : Dune_project.t
+    ; loc : Loc.t
+    ; old_name : 'old_name
+    ; new_public_name : Loc.t * Lib_name.t
+    }
+
+  module Private : sig
```

Comment:
Should we call this "Local", by analogy with `Lib_name.Local`?

Reply:
That sounds better indeed.